RNAseq_SR (#126)
galaxyproject/iwc: workflows/transcriptomics/RNAseq_SR/README.md

# RNA-seq single-read Workflow

## Inputs dataset

- The workflow needs a list of dataset of fastqsanger.
- As well as a gtf file with genes

> - And a gtf file with regions to exclude from normalization in Cufflinks.

Reviewer: ```suggestion
- Optional, but recommended: a gtf file with regions to exclude from normalization in Cufflinks.
```

  - You can either put an empty file of datatype gtf

>   - Or provide a gtf with usually chrM, for example for mm10:

Reviewer: ```suggestion
  -For instance a gtf that masks chrM for the mm10 genome:
```
